policyengine_us/variables/gov/states/mt/tax/income/deductions/standard/mt_standard_deduction.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class mt_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana standard deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        itemizing = tax_unit("tax_unit_itemizes", period)
```

Comment:
Remove this condition - we check the itemization choice in a later step in the computation tree so we can optimize the filer's choice.